src/includes/classes/Plugin.php

```diff
@@ -490,6 +490,12 @@ public function setup()
         /*[/pro]*/
 
         /*[pro strip-from="lite"]*/
+        if ($this->options['when_logged_in'] === '1' && $this->applyWpFilters(GLOBAL_NS.'_when_logged_in_no_admin_bar', true)) {
+          show_admin_bar(FALSE); // Prevent admin bar from being cached.
```

Comment:
@raamdev I think the docs on that `show_admin_bar()` function are a little outdated now. I just worked with trying to hide the admin bar whenever I built the demo.s2member.com site. I had to jump through some hoops in order to hide it in the Dashboard. It seems they have made it impossible to hide the admin bar in the Dashboard with this function or with the filter. The only way I was able to hide it in the Dashboard was by removing the rendering hook and by also injecting some custom CSS into the head.

Referencing: https://github.com/websharks/demos.websharks-inc.com/blob/master/wp-content/mu-plugins/demo.s2member.com.php#L39

Referencing: https://github.com/websharks/demos.websharks-inc.com/blob/master/wp-content/mu-plugins/demo.s2member.com.php#L161